IFComp/root/src/_current_entry_row.tt

```diff
@@ -77,6 +85,7 @@
               || entry.platform == 'quixe'
               || entry.platform == 'quest-online'
               || entry.platform == 'inform'
+              || entry.platform == 'tads-web-ui'
```

Comment:
Perhaps it would be better for this to be a derived property on the model, but that's a change for another day.